config.yaml

```diff
@@ -228,6 +228,12 @@ postgresql_server:
   # THE PASSWORD WILL BE REMOVED FROM THE FILE AFTER THE INSTALLATION FINISHES
   postgres_password: ''
 
+  # postgresql.conf overrides
+  config:
+    shared_buffers: '1024MB'
+    effective_cache_size: '1024MB'
+    work_mem: '16MB'
```

Comment:
16MB * the 20 or so connections that we have = postgres itself shouldn't go above 500ish MB of actual usage (depending on the query, a single conn. might use several times the work_mem, but unlikely)